app/Services/FlutterwaveService.php

```
@@ -0,0 +1,77 @@
+    {
+        $response = Http::withHeaders([
+            'Authorization' => 'Bearer ' . $this->secretKey,
+        ])->get("{$this->baseUrl}/transactions/{$reference}/verify");
+
+        return $response->json();
+    }
+
+    /**
+     * Validate webhook signature
+     */
+    public function validateWebhookHash(array $payload): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$payload'. (Unused Code Rules)

(UnusedFormalParameter)